src/month_calendar/utils.es

```diff
@@ -0,0 +1,41 @@
+import addMonths from 'date-fns/addMonths';
+import getDaysInMonth from 'date-fns/getDaysInMonth';
+import formatDate from 'date-fns/format';
+
+export const getMonth = (string) => {
+  const [year, month] = string.split('-').map((item) => parseInt(item, 10));
+  return new Date(year, month - 1);
+};
+
+export const getMonthString = (current, diff = 0) => (
```

Comment:
i don't think we need this at all, do we?

Reply:
Oh yeah we need all this. I already got rid of the unused stuff. `getMonthString` is used to navigate months (see `month_calendar/index.es`)